tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/SpecWorkflowTool/SpecWorkFlowTool.cs

```diff
@@ -465,16 +457,10 @@ public async Task<string> LinkSdkPullRequestToReleasePlan(string language, strin
                     return $"Release plan with ID {releasePlanId} or work item ID {workItemId} is not found.";
                 }
 
-                await devopsService.AddSdkInfoInReleasePlanAsync(releasePlan.WorkItemId, language, "", parsedLink);
+                var sdkInfoInRelease = devopsService.AddSdkInfoInReleasePlanAsync(releasePlan.WorkItemId, language, "", parsedLink);
```

Comment:
(outsider comment)

It's so weird, based on the names of these methods, that they don't have sort of ordering dependency.

Reply:
Yes. I think they shouldn't depend on each other. They use similar data but they are not the same. If one fails, I would expect the tool to retry it. I won't expect the tool to fail for both.
I don't think one has priority over the other one either